google/datalab/contrib/bigquery/operators/_bq_load_operator.py

```diff
@@ -67,3 +65,7 @@ def execute(self, context):
       raise Exception('Load failed: %s' % str(job.fatal_error))
     elif job.errors:
       raise Exception('Load completed with errors: %s' % str(job.errors))
+
+    return {
+      'result': job.result()
```

Comment:
Same thing here. Is job.result a simple type?
And shouldn't this also return the table into which data was loaded?